x-pack/plugin/sql/qa/src/main/resources/null.csv-spec

```diff
@@ -0,0 +1,60 @@
+//
+// Null expressions
+//
+
+dateTimeOverNull
+SELECT YEAR(CAST(NULL AS DATE)) d;
+
+d:i
+null
+;
+
+addOfNull
+SELECT CAST(NULL AS INT) + CAST(NULL AS FLOAT) AS n;
+
+n:d
+null
+;
+
+
+divOfNull
+SELECT 5 / CAST(NULL AS FLOAT) + 10 AS n;
+
+n:d
+null
+;
+
+coalesceJustWithNull
+SELECT COALESCE(null, null, null) AS c;
+
+c
+null
+;
+
+coalesceWithFirstNullOfString
+SELECT COALESCE(null, 'first') AS c;
+
+c:s
+first
+;
+
+coalesceWithFirstNullOfNumber
+SELECT COALESCE(null, 123) AS c;
+
```

Comment:
I would also add a test where all values are `null`.

Reply:
it's done here: https://github.com/elastic/elasticsearch/pull/35253/files#diff-4fb3d311239a963fa6e56c5db4264130R28 no?

Reply:
Hm... my bad, not sure why I haven't seen that.